### PR TITLE
R conditionals

### PR DESCRIPTION
**Pull request by jdagz28**
### Description
Added information on conditionals in R at docs/content/r/concepts/conditionals/conditionals.md.

### Type of Change
- Adding a new entry

### Checklist
- [✅] All writings are my own.
- [✅] My entry follows the Codecademy Docs style guide.
- [✅] My changes generate no new warnings.
- [✅] I have performed a self-review of my own writing and code.
- [✅] I have checked my entry and corrected any misspellings.
- [✅] I have made corresponding changes to the documentation if needed.
- [✅] I have confirmed my changes are not being pushed from my forked `main` branch.
- [✅] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [✅] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/r/concepts/conditionals/conditionals.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+    x <- 5
+    y <- 10
+    
+    if (x < y) {
+      print("x is less than y")
+    }
+    
+    ```
+    
+    Output: `x is less than y`
```
Comment (caupolicandiaz) [inline content/r/concepts/conditionals/conditionals.md:74]: Usually we prefer using a separate shell block for displaying the output; however, with entries such as this, with many examples, we allow for commented outputs within the same block.

File: content/r/concepts/conditionals/conditionals.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+
+```
+
+Output: `x is greater than or equal to y`
+
+These operators are commonly used in conditional statements to compare values and make decisions.
+
+## Logical Operators
+
+Logical operators in R allow you to combine or negate logical values. Here are the commonly used logical operators:
```
Comment (caupolicandiaz) [inline content/r/concepts/conditionals/conditionals.md:163]:
```suggestion
Logical operators in R allow can be used to combine or negate logical values. Here are the commonly used logical operators:
```

File: content/r/concepts/conditionals/conditionals.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+
+- `&&` (and): Returns `TRUE` if both the left and right operands are `TRUE`.
```
Comment (caupolicandiaz) [inline content/r/concepts/conditionals/conditionals.md:165]: I think this section should also be consolidated in two parts as above. List all of the operators with their definitions and then the examples.

File: content/r/concepts/conditionals/conditionals.md [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+    }
+    
+    ```
+    
+    Output: `x is not equal to 10`
+    
+
+Logical operators are often used to combine multiple conditions in conditional statements or to negate a condition.
+
+These are the basic conditional statements, comparison operators, and logical operators in R. They allow you to control the flow of your program, compare values, and make decisions based on specific conditions.
```
Comment (caupolicandiaz) [inline content/r/concepts/conditionals/conditionals.md:215]:
```suggestion
These are the basic conditional statements, comparison operators, and logical operators in R. They can be leveraged to control the flow of a program, compare values, and make decisions based on specific conditions.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@jdagz28 I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by jdagz28** [comment]
Hi @caupolicandiaz, made the necessary changes. Please let me know if there are more changes to be made.

Thank you.

File: content/r/concepts/conditionals/conditionals.md [hunk 5]
```diff
@@ -1,215 +1,181 @@
-# R Conditionals
-
-Conditional statements in R allow you to control the flow of your program based on certain conditions. You can execute different blocks of code depending on whether a condition is true or false.
+---
+Title: 'R - Conditionals'
```
Comment (caupolicandiaz) [inline content/r/concepts/conditionals/conditionals.md:2]:
```suggestion
Title: 'Conditionals'
```

File: content/r/concepts/conditionals/conditionals.md [hunk 6]
```diff
@@ -1,215 +1,181 @@
...
+Description: 'A brief description on the usage of conditional statements, comparison and logic operators in R'
```
Comment (caupolicandiaz) [inline content/r/concepts/conditionals/conditionals.md:3]:
```suggestion
Description: 'Statements that return a boolean value that provide control flow in a program.'
```

File: content/r/concepts/conditionals/conditionals.md [hunk 7]
```diff
@@ -1,215 +1,181 @@
...
+Subjects:
+  - 'Conditinal Statements'
```
Comment (caupolicandiaz) [inline content/r/concepts/conditionals/conditionals.md:5]:
```suggestion
  - 'Conditional Statements'
```

File: content/r/concepts/conditionals/conditionals.md [hunk 8]
```diff
@@ -1,215 +1,181 @@
...
+  - 'Comparison Operators'
+  - 'Logical Operators'
+Tags:
+  - 'R'
+  - 'conditionals'
```
Comment (caupolicandiaz) [inline content/r/concepts/conditionals/conditionals.md:10]:
```suggestion
  - 'Conditionals'
```

File: content/r/concepts/conditionals/conditionals.md [hunk 9]
```diff
@@ -1,215 +1,181 @@
...
+  - 'comparison'
```
Comment (caupolicandiaz) [inline content/r/concepts/conditionals/conditionals.md:11]:
```suggestion
  - 'Comparison'
```

File: content/r/concepts/conditionals/conditionals.md [hunk 10]
```diff
@@ -1,215 +1,181 @@
...
+  - 'logic'
```
Comment (caupolicandiaz) [inline content/r/concepts/conditionals/conditionals.md:12]:
```suggestion
  - 'Logic'
```

File: content/r/concepts/conditionals/conditionals.md [hunk 11]
```diff
@@ -1,215 +1,181 @@
...
+
```
Comment (caupolicandiaz) [inline content/r/concepts/conditionals/conditionals.md:13]:
```suggestion
```

File: content/r/concepts/conditionals/conditionals.md [hunk 12]
```diff
@@ -1,215 +1,181 @@
...
 - `&&` (and): Returns `TRUE` if both the left and right operands are `TRUE`.
-    
-    Example:
-    
-    ```r
+- `||` (or): Returns `TRUE` if either the left or right operand is `TRUE`.
+- `!` (not): Negates a logical value. If the operand is `TRUE`, it returns `FALSE`, and if the operand is `FALSE`, it returns `TRUE`.
+
+Examples:
+  ```R
```
Comment (caupolicandiaz) [inline content/r/concepts/conditionals/conditionals.md:149]:
```suggestion
```r
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@jdagz28 I've added some additional comments. Tag me for another review when you've made the changes. Thanks

File: content/r/concepts/conditionals/conditionals.md [hunk 13]
```diff
@@ -0,0 +1,182 @@
+---
+Title: 'Conditionals'
+Description: 'Statements that return a boolean value that provide control flow in a program'
```
Comment (SSwiniarski) [inline content/r/concepts/conditionals/conditionals.md:3]:
```suggestion
Description: 'Conditional statements return a boolean value that provides control flow in a program.'
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@jdagz28 I did a review and left you a comment.

**Comment by jdagz28** [comment]
@SSwiniarski edited the description already.

**Comment by SSwiniarski** [comment]
@jdagz28 It's up to you. We can merge without that e-mail signing the CLA, but it may have effects on you getting credit for your entry. If you don't mind an anonymous entry, we can go ahead with this PR. 

Otherwise, you can make a new PR (using your GitHub e-mail) with the edited entry in its current form. On that PR request a review from me and @caupolicandiaz and leave a comment on the new entry tagging both of us, letting us know we already reviewed it. Then you can close this one.

**Comment by jdagz28** [comment]
@SSwiniarski  I am okay with an anonymous entry for this pull request.  Please go ahead with the merge. Thank you very much.

**Review by SSwiniarski — APPROVED** [review]
@jdagz28 Okay, looks like we can merge!

**Comment by SSwiniarski** [comment]
@jdagz28 congrats on the new entry! The live link is here: https://www.codecademy.com/resources/docs/r/conditionals

image

As I said, this is anonymous. If you want credit on future entries, makes sure that you use your GitHub email is used to commit the PR, and that your GitHub account is connected to your Codecademy profile.

**Pull request merged**